.github/ISSUE_TEMPLATES/bug_report.yml

```diff
@@ -0,0 +1,62 @@
+---
+name: Bug Report
+description: Create a Report to Help us Improve
+title: "[BUG]: "
+labels:
+  - bug
+assignees:
+  - paddyroddy
+  - samcunliffe
+body:
+  - id: describe
+    type: textarea
+    attributes:
+      label: Describe the Bug
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - id: reproduce
+    type: textarea
+    attributes:
+      label: To Reproduce
+      description: >-
+        A minimal working example of code to reproduce the unexpected behaviour.
+    validations:
+      required: true
+  - id: expected
+    type: textarea
+    attributes:
+      label: Expected Behaviour
+      description: >-
+        A clear and concise description of what you expected to happen.
+    validations:
+      required: true
+  - id: actual
+    type: textarea
+    attributes:
+      label: Actual Behaviour
+      description: >-
+        Be a specific and detailed as you can.
+        Paste any output or stack traces of errors you receive.
+    validations:
+      required: true
+  # - id: version
+  #   type: input
+  #   attributes:
+  #     label: Version In Use
+  #     description: |-
+  #       Can be found by
+  #       ```sh
+  #       python -c "import sleplet; print(sleplet.__version__)"
+  #       ```
+  #   validations:
+  #     required: true
```

Comment:
I'm going to modify this in a different PR once we get the tagging thing working